src/openforms/registrations/api/views.py

```diff
@@ -33,6 +35,27 @@ def get_objects(self):
         return list(register)
 
 
+@extend_schema_view(
+    get=extend_schema(
+        summary=_("Get the json schema of the plugins configuration options")
+    ),
+)
+class PluginsConfigurationOptionsView(RetrieveAPIView):
```

Comment:
There is zero reason to use `RetrieveAPIView` which is a generic, model-based API view. We're not dealing with models in the slightest here and the `get` method is completely custom anyway. This should just be `APIView`.